shared/src/business/utilities/DateHandler.js

```diff
@@ -13,6 +13,7 @@ const FORMATS = {
   MMDDYYYY_DASHED: 'MM-dd-yyyy',
   MONTH_DAY_YEAR: 'MMMM d, yyyy',
   MONTH_DAY_YEAR_WITH_DAY_OF_WEEK: 'DDDD',
+  SHORT_MONTH_DAY_YEAR: 'MMM d, yyyy',
```

Comment:
what's the reason for adding this? is this in the AC or design? maybe i missed it somewhere

Reply:
@rosiehsmith 
This was added to save spaces in the pdf header. We found out there wasn't enough space for the combination of the longest edge cases like a spelled out city (e.g Washington, District of Columbia); spelled out date (December 25th, 2022); Judges name (Gustafson) and page number. 

After a conversation with the UI/UX team, the best decision then was to have a short version of the date to accommodate for the long edge cases.

Reply:
ah i see, thanks josh. seems like there might be a benefit to having that documented somewhere, whatcha think @liadmccabe?